PermanentOrg/web-app
# Per 9494 convert mixpanel calls

This pull request replaces the previous mixpanel tracking from the web-app with the back-end tracking using api's

Steps to test:
1. Open the network tab
2. Complete any action which should be tracked by mixpanel
3. In the network tab there should be an event with the name **event** which should send a good result
